src/app/error.tsx

```
@@ -0,0 +1,36 @@
+"use client";
+
+import { useEffect } from "react";
+
+export default function Error({
```

Biome (2.1.2)

[error] Do not shadow the global "Error" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)